snappiershot/serializers/json.py

```diff
@@ -205,6 +222,40 @@ def encode_collection(value: Collection) -> JsonType:
             f"No encoding implemented for the following collection type: {value} ({type(value)})"
         )
 
+    @staticmethod
+    def encode_pandas(value: Any) -> JsonType:
+        """ Encoding for collection types.
+
+        The custom encoding follows the template:
+            {
+              PANDAS_KEY: <type, as a string>>,
+              PANDAS_VALUE_KEY: <value, with a list or dict-of-lists with (index, value) tuples>
+            }
+
+        The values for the PANDAS_KEY and PANDAS_VALUE_KEY constants are attributes
+          to the `snappiershot.serializers.constants.CustomEncodedPandasTypes` class.
+
+        Raises:
+            NotImplementedError - If encoding is not implemented for the given pandas type.
+        """
+        # A special effort is made to avoid importing pandas unless it's really necessary.
+        import pandas as pd
```

Comment:
I don't love that we are importing pandas here, especially since we are only using it for types. Ideally I would like to see us be able to write this function without having to explicitly import pandas like this. Maybe we use get_pandas helper method and raise an explicit error?

Reply:
done